pkgs/shells/gash/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchurl
+, pkg-config
+, guile_3_0
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gash";
+  version = "0.3.0";
+
+  src = fetchurl {
+    url = "mirror://savannah/gash/gash-${finalAttrs.version}.tar.gz";
+    hash = "sha256-VGrsaRBo1nfFjd/BVpXbn4CGFuGfpzMi1Ppno8iXwqk=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
```

```suggestion
  nativeBuildInputs = [
    pkg-config
    guile_3_0
  ];
```

Necessary for cross compilation.